### PR TITLE
add enable/disable button

### PR DESCRIPTION
**Pull request by tuplink**
Fixes https://github.com/pi-hole/pi-hole/issues/449.

Changes proposed in this pull request:
-updates status from pihole script
-provides a button to enable and disable
depends on https://github.com/pi-hole/pi-hole/pull/814 and https://github.com/pi-hole/pi-hole/pull/823
@pi-hole/dashboard

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -1,4 +1,13 @@
 <?php
+	if (isset($_GET['enable'])) {
+	  exec('sudo pihole enable');
+	  $refer = $_SERVER['HTTP_REFERER'];
+	  header("location:$refer");
+	} elseif (isset($_GET['disable'])) {
+	  exec('sudo pihole disable');
+	  $refer = $_SERVER['HTTP_REFERER'];
+	  header("location:$refer");
+	}
```
Comment (Mcat12) [inline header.php]: Please convert to spaces.

Comment (tuplink) [inline header.php]: ✔️

**Comment by tuplink** [comment]
@Mcat12 have a look now   added a button in the menu

**Comment by Mcat12** [comment]
If you would like, you could split out some of this code into `api.php` so that when you click the button it simply does an ajax call to the api. This does work fine in it's current form, but an ajax call would reduce load on the Pi-hole by not having to refresh all the graphs and such. I'm not going to require you to restructure this PR, but it's a good idea.

**Comment by Mcat12** [comment]
Approved

**Comment by brantje** [comment]
Code looks good :+1: 
Approved

**Comment by PromoFaux** [comment]
Approved!

Can be merged when https://github.com/pi-hole/pi-hole/pull/814 is merged to development

**Pull request merged**